### PR TITLE
Make VSO user in seed data functional

### PR DESCRIPTION
**Pull request by lowellrex**
@nikitarockz discovered that our VSO user in the test data does not actually have access to the VSO queue. This PR fixes that.

### CODE REVIEW

File: db/seeds.rb [hunk 1]
```diff
@@ -44,7 +44,14 @@ def create_users
 
     Functions.grant!("System Admin", users: User.all.pluck(:css_id))
 
-    User.create(css_id: "VSO", station_id: 101, full_name: "VSO user associated with american-legion")
+    u = User.create(
+      css_id: "VSO",
+      station_id: 101,
+      full_name: "VSO user associated with american-legion",
+      roles: ["VSO"]
```
Comment (lowellrex) [inline db/seeds.rb]: Add role to pass [`OrganizationsController.verify_role_access()`](https://github.com/department-of-veterans-affairs/caseflow/blob/2fc9a9ac7681c6475b70d1ae58943c4cfea63592/app/controllers/organizations_controller.rb#L17) check.

File: db/seeds.rb [hunk 2]
```diff
@@ -44,7 +44,14 @@ def create_users
...
+    )
+    FeatureToggle.enable!(:vso_queue_aml, users: [u.css_id])
```
Comment (lowellrex) [inline db/seeds.rb]: Enable feature to pass [`OrganizationsController. verify_feature_access()`](https://github.com/department-of-veterans-affairs/caseflow/blob/2fc9a9ac7681c6475b70d1ae58943c4cfea63592/app/controllers/organizations_controller.rb#L21) check.

File: db/seeds.rb [hunk 3]
```diff
@@ -44,7 +44,14 @@ def create_users
...
+    Functions.grant!("VSO", users: [u.css_id])
```
Comment (lowellrex) [inline db/seeds.rb]: Grant function to pass [`ApplicationController. can_access_queue?()`](https://github.com/department-of-veterans-affairs/caseflow/blob/2fc9a9ac7681c6475b70d1ae58943c4cfea63592/app/controllers/application_controller.rb#L97) check.

Comment (mdbenjam) [reply db/seeds.rb]: So I don't think we want to use functions. I was noticing that earlier in the code. I think https://github.com/department-of-veterans-affairs/caseflow/blob/2fc9a9ac7681c6475b70d1ae58943c4cfea63592/app/models/user.rb#L133 this line should be `roles.include?("VSO")`

File: db/seeds.rb (hunk 1 above)
Comment (mdbenjam) [reply db/seeds.rb]: Weird, thought we already had a role defined. Don't know how this was ever working. Maybe it wasn't...

File: db/seeds.rb (hunk 3 above)
Comment (lowellrex) [reply db/seeds.rb]: The difference being that Functions come from CSEM and roles come from VACOLS?

File: db/seeds.rb (hunk 1 above)
Comment (lowellrex) [reply db/seeds.rb]: Not sure how it was either.

File: db/seeds.rb (hunk 3 above)
Comment (lowellrex) [reply db/seeds.rb]: Ahhhh gotcha. I see what you're saying.

**Pull request merged**